x-pack/plugins/siem/public/components/page/hosts/events_table/translations.ts

```diff
@@ -10,6 +10,14 @@ export const EVENTS = i18n.translate('xpack.siem.eventsTable.eventsTitle', {
   defaultMessage: 'Events',
 });
 
+export const UNIT_SINGULAR = i18n.translate('xpack.siem.eventsTable.unitSingular', {
+  defaultMessage: 'Event',
+});
+
+export const UNIT_PLURAL = i18n.translate('xpack.siem.eventsTable.unitPlural', {
+  defaultMessage: 'Events',
+});
+
```

Comment:
This might be helpful?

Optional, but if you look at the unit tests for `i18n`, you can use a plural for it.
https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/src/core/i18n.test.ts#L516

So if you have to use a string because of an existing API and cannot use a ReactElement, you shouldn't lose the ability to do plurals, but please correct my understanding if it's wrong (I only speak one language so might not be the best person to give i18n advice).

Reply:
An example where I use it is with rows:

```ts
export const ROWS_5 = i18n.translate('xpack.siem.authenticationsTable.rows', {
  values: { numRows: 5 },
  defaultMessage: '{numRows} {numRows, plural, =0 {rows} =1 {row} other {rows}}',
});
```

Reply:
Think I got it working.